index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <script src="https://kit.fontawesome.com/38b6d4ab77.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="/css/styles.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@100&display=swap"
+    rel="stylesheet" />
+  <title>Zunair Mobile Setup Website</title>
+</head>
+
+<body>
+  <div class="container">
+    <section class="header-section">
+      <nav class="nav-bar">
+        <a class="logo">Zunair khan</a>
+        <i class="fa fa-bars menu-bar" aria-hidden="true"></i>
+      </nav>
+    </section>
```

Comment:
The previous reviewer requested changes for line 25. That needs to be `<section>` but your nav line 19-24 needs to be wrapped within `<header>` element. It is more correct for [semantic](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) reasons. Please fix it.

Reply:
done